services/api_service/api_service/views.py

```diff
@@ -1,10 +1,35 @@
+from api_service.pubsub import PubSubManager
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
+import random
+import json
+import time
+import threading
+import uuid
+from api_service.result_watcher import ResultWatcher
 
 
+SUBSCRIPTIONS_PATH = "projects/lohfinder-app/subscriptions/"
+PROJECT_ID = "lohfinder-app"
+pubsub_manager = PubSubManager(SUBSCRIPTIONS_PATH, PROJECT_ID)
+users_result_watcher = ResultWatcher(pubsub_manager, "user_service_result-sub", lambda m : m.attributes["message_id"])
+
+def get_id():
+    return str(uuid.uuid4().int & (1<<64)-1)
```

Comment:
we need to set seed depending on time here, otherwise numbers will be the same with every execution

Reply:
uuid.uuid4 generates new numbers each time. Checked it on two different terminals